<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addpart</class>
 <widget class="QDialog" name="addpart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Peça</string>
  </property>
  <widget class="QPushButton" name="btn_Cadastrar">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>280</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Cadastrar!</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Emoticon">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>170</y>
     <width>71</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_NomeDaPeca">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>41</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Feedback">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Insira os dados da peça e clique em Cadastrar.</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Cidade">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>41</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Custo</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Descricao">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Descrição</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Quantidade">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantidade</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_Nome">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>50</number>
   </property>
   <property name="placeholderText">
    <string>Nome da Peça. Exemplo: Escape Corsa 98</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Sair">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>280</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_Quantidade">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinCusto">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="singleStep">
    <double>5.000000000000000</double>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txt_PartDescription">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>381</width>
     <height>111</height>
    </rect>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Máximo de 100 letras.</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>line_Nome</tabstop>
  <tabstop>txt_PartDescription</tabstop>
  <tabstop>doubleSpinCusto</tabstop>
  <tabstop>spin_Quantidade</tabstop>
  <tabstop>btn_Cadastrar</tabstop>
  <tabstop>btn_Sair</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
